src/transactions/typeTag/typeTag.ts

```diff
@@ -0,0 +1,380 @@
+// Copyright © Aptos Foundation
```

Comment:
I think this file has been renamed

Reply:
I changed it to match the other files, since everything else uses `index.ts`, greg said just get it in so just gonna leave it in